### DISCUSSION TITLE
ask about update and reload liquidsoap config file without kill process

### DISCUSSION THREAD

**Discussion by wezo3** [savonet/liquidsoap#2793] [General]
Hello,
there any way can i do to update liquidsoap config file without kill the current process ?
for example i have this process already running 
root     1856994 11.7 10.7 1172772 423700 ?      Sl   Nov21 4712:58 /bin/liquidsoap -f /root/config/liquidsoap.liq

and updated the config file to change any configuration and need to apply this without kill process to not cut current track or streaming

there any hint to do this to let liquidsoap reload the configuration again in same process ?
Thank you

**Comment by smimram** [comment]
Not currently (or in any forseeable future).

However things might be done depending on your actual situation: if you want to change parameters from time to time, or reload a playlist, there are certainly ways to do this.

**Reply by wezo3** [reply]
thank you @smimram for your response
what this other ways for example you mean ?
for example if I want to change password or port in input.harbor settings or add more input.harbor settings or update crossfade values setting how can do this by other ways ?
Thank you for help
